templates/core/dataset/item-list--dg-auth-block-menu.html.twig

```diff
@@ -25,17 +25,21 @@
 {% endif %}
 
 {% if items or empty -%}
-  {%- if items -%}
-    {%- for item in items -%}
-      {% if loop.last %}
-        <footer>
-          <a href="{{ item.value.value['#url'] }}" {{ item.attributes }}>{{ item.value.value['#title'] }}</a>
-        </footer>
-      {% else %}
-        {{ item.value }}
-      {% endif %}
-    {%- endfor -%}
-  {%- else -%}
-    {{- empty -}}
-  {%- endif -%}
+  <div {{ wrapper_attributes.addClass('item-list') }}>
+    {%- if items -%}
+      {%- for item in items -%}
+        {% if loop.last %}
+          <footer>
+            <a href="{{ item.value.value['#url'] }}" {{ item.attributes }}>{{ item.value.value['#title'] }}</a>
```

Comment:
@matthijs-va, de `item.attributes` wordt wel geprint op de a-tag (in plaats van de footer), dat wijkt inderdaad af van de andere items, maar aangezien die link manueel wordt opgebouwd in de template lijkt me dat geen probleem. 

Tenzij jij daar anders over denkt?